src/index.js

```diff
@@ -0,0 +1,161 @@
+    state.decreaseTempControl.addEventListener("click", displayLandscape);
+    state.decreaseTempControl.addEventListener("click", handleDecreaseTempClick);
+    state.currentTempButton.addEventListener("click", handleCurrentTempClick);
+    state.skySelection.addEventListener("change", displaySky);
+    state.cityNameInput.addEventListener("input", changeCity);
+    state.resetButton.addEventListener("click", resetCityName);
+};
+
+const handleCurrentTempClick = (event) => {
+    displayLandscape();
+    temperatureColor();
+    findLatitudeAndLongitude(state.cityNameInput.value);
+};
+
+const handleIncreaseTempClick = (event) => {
+    // When clicking on the increase temp button, the temp value should increase by 1
+    state.tempCount += 1;
+    state.tempValue.textContent = state.tempCount;
+    temperatureColor();
+};
+
+const handleDecreaseTempClick = (event) => {
+    // When clicking on the decrease temp button, the temp value should decrease by 1
+    state.tempCount -= 1;
+    state.tempValue.textContent = state.tempCount;
+    temperatureColor();
+};
+
+const displayLandscape = (event) => {
+    if (state.tempCount < 59) {
+        state.landscapeDiv.textContent = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+    } else if (state.tempCount <= 69) {
+        state.landscapeDiv.textContent = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+    } else if (state.tempCount <= 79) {
+        state.landscapeDiv.textContent = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+    } else if (state.tempCount >= 80) {
+        state.landscapeDiv.textContent = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+}};
+
+const colorChange = (color) => {
+    document.getElementById("tempValue").style.color = color
+};
+
+const temperatureColor = (event) => {
+    const temp = state.tempCount
+    if (temp <= 49) {
+        colorChange("teal");
+    } else if (temp <= 59) {
+        colorChange("green");
+    } else if (temp <= 69) {
+        colorChange("yellow");
+    } else if (temp <= 79) {
+        colorChange("orange");
+    } else {
+        colorChange("red");
+    }
+};
+
+const displaySky = (event) => {
+    if (state.skySelection.value === "sunny") {
+        state.skyDiv.textContent = "☀️☀️☀️☀️☀️☀️";
+    } else if (state.skySelection.value === "cloudy") {
+        state.skyDiv.textContent = "☁️☁️⛅☁️☁️☁️";
+    } else if (state.skySelection.value === "rainy") {
+        state.skyDiv.textContent = "☁️☔️☁️☁️☔️☁️";
+    } else if (state.skySelection.value === "snowy") {
+        state.skyDiv.textContent = "☁️❄️☁️❄️☁️❄️";
+    }
+};
+
+const changeCity = (event) => {
+    const value = state.cityNameInput.value;
+    state.cityNameHeader.textContent = value;
+};
+
+const resetCityName = (event) => {
+    state.cityNameInput.value = "";
+    state.cityNameHeader.textContent = "";
+};
+
+const findLatitudeAndLongitude = (cityName) => {
+    axios.get('http://127.0.0.1:5000/location',
+    { 
+    params: {
+        q: cityName
+    }
+    })
+    .then( (response) => {
+        console.log(response.data)
```

Comment:
Logging to the console is a very useful debugging technique, however, we don't want to end up merging code in that contains log statements, so be sure to check your code before submitting a PR to make sure you've removed any debugging console logs.